staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -15,9 +15,36 @@
 package model
 
 import (
+	"time"
+
+	"github.com/google/cel-go/checker/decls"
+	"github.com/google/cel-go/common/types"
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 )
 
+const (
+	// the largest request that will be accepted is 3MB
+	// TODO(DangerOnTheRanger): find out where this is originally declared so we don't have two constants
```

Comment:
https://github.com/kubernetes/kubernetes/blob/2cd8ceb2694ef30d93cccb53445e9add6cbd9f7f/staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go#L55

https://github.com/kubernetes/kubernetes/blob/e1c16d24a101188d9ce6ffe8432de6ba60ab5efb/staging/src/k8s.io/apiserver/pkg/server/config.go#L362

Reply:
Oh, thanks - I'll use that constant instead.

Reply:
You won't be allowed to reference it because of where it is defined. Ideally we'd wire it in. I don't think you should try to figure that out right now though. So go ahead and copy over the `int64(3 * 1024 * 1024)` and then leave a TODO to wire in the config MaxRequestBodyBytes to make this configurable on with a non-default value (it's not configurable via flag, so anyone changing this has to compile their own apiserver).

Reply:
Gotcha, fixed